installation/multi_machine_aws.rst

```diff
@@ -41,7 +41,7 @@ Below, we explain in detail the steps required to setup a multi-node Citus clust
 
 .. raw:: html 
   
-  <p>To begin, you can start a Citus cluster using CloudFormation by clicking <a href="https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=Citus&templateURL=https:%2F%2Fcitus-deployment.s3.amazonaws.com%2Faws%2Fcitus6%2Fcloudformation%2Fcitus-6.1.0.json" onclick="trackOutboundLink('https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=Citus&templateURL=https:%2F%2Fcitus-deployment.s3.amazonaws.com%2Faws%2Fcitus6%2Fcloudformation%2Fcitus-6.1.0.json'); return false;">here</a>. This will take you directly to the AWS CloudFormation console.</p>
+<p>To begin, you can start a Citus cluster using CloudFormation by clicking <a href="https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=Citus&templateURL=https:%2F%2Fcitus-deployment.s3.amazonaws.com%2Faws%2Fcitus6%2Fcloudformation%2Fcitus-6.2.0.json" onclick="trackOutboundLink('https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=Citus&templateURL=https:%2F%2Fcitus-deployment.s3.amazonaws.com%2Faws%2Fcitus6%2Fcloudformation%2Fcitus-6.2.0.json'); return false;">here</a>. This will take you directly to the AWS CloudFormation console.</p>
```

Comment:
The CF template itself hasn't been updated yet. @jasonmp85 @byucesoy I presume this will happen after packaging. We can then copy this over to S3 and update this link.